BuildingSync.xsd

```diff
@@ -4030,7 +4030,7 @@
       <xs:element ref="auc:ResourceBoundary" minOccurs="0"/>
       <xs:element name="SiteEnergyUse" minOccurs="0">
         <xs:annotation>
-          <xs:documentation>The annual amount of all the energy the premises consumes onsite, as reported on the utility bills. (kBtu)</xs:documentation>
+          <xs:documentation>The annual amount of all the energy the premises consumes onsite, as reported on the utility bills. Calculated as imported energy - exported energy - net increase in stored energy (per ASHRAE 105-2014 Figure 5.6). (kBtu)</xs:documentation>
```

Comment:
@corymosiman12 can you confirm this is the right "name" of the standard?

Reply:
Yes, standard name is correct.

Change doc wording to match:
`Calculated as imported energy (Eimp) - exported energy (Eexp) - net increase in stored imported energy (Es)`